docs/help.md

```diff
@@ -21,4 +21,4 @@ If you want to report a bug:
 - be sure to first read about [how to contribute](https://github.com/docker/distribution/blob/master/CONTRIBUTING.md)
 - you can then do so on the [GitHub project bugtracker](https://github.com/docker/distribution/issues)
 
-You can also find out more about the Docker's project [Getting Help resources](https://docs.docker.com/opensource/get-help/).
+You can also find out more about the Docker's project [Getting Help resources](/opensource/get-help.md).
```

Comment:
Downside here is that these will be broken on GitHub. Perhaps we should think more about how to do cross-repo linking. I know we discussed making these links to the GitHub repository and rewrite them to relative links during the docs build.